docs/cms-adapters/CMS_ADAPTERS.md

```diff
@@ -63,6 +63,44 @@ For an up-to-date documentation, run `ng help @o3r/components:extractor`
 
 __Note:__ This options will not search for the duplicate configurations in libraries.
 
+#### How to check breaking changes
+
+Version after version, it can be verified that any breaking changes has been introduced,
```

Comment:
```suggestion
Version after version, it can be verified whether any breaking changes have been introduced, or if there is any metadata provided to document it.
```